CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Guia de contribuição
+
+## Antes de começar
+
+Ficamos muito felizes que você está lendo este guia de contribuição, sempre precisamos de desenvolvedores voluntários que acreditem na ideia e queiram contribuir com o projeto.
+
+Se você ainda não fez isso, junte-se também a nós [nosso grupo aberto do Telegram](https://t.me/joinchat/DRT0JBcy-RUk2GJZCnH3Fg) e participe das discussões. Não hesite em nos procurar para tirar todas as suas dúvidas.
+
+Aqui estão alguns importantes recursos que você deve estar ciente antes de começar:
+
+- [Manual de dados abertos para desenvolvedores](https://www.w3c.br/pub/Materiais/PublicacoesW3C/manual_dados_abertos_desenvolvedores_web.pdf) te explicará um pouco sobre o que são dados abertos.
+
+Não possuímos um roadmap público, mas você pode acompanhar o que a comunidade vem trabalhando através do [nosso Trello](https://trello.com/b/E8v20MFs/dados-abertos-de-feira)
+
+Os detalhes de como instalar e executar este projeto podem ser encontrados no [README.md](https://github.com/DadosAbertosDeFeira/maria-quiteria/blob/master/README.md)
+
+## Reportando bugs
+
+Você encontrou um bug ?
+Esta seção irá te guiar sobre como você deve agir para reportar um Bug no Maria Quitéria. Seguir estas etapas ajuda os mantenedores do projeto e a comunidade a entender o que aconteceu, reproduzir o erro e encontrar a solução.
+
+- Comunique o bug encontrado no no nosso [grupo no Telegram](https://t.me/joinchat/DRT0JBcy-RUk2GJZCnH3Fg) com outros participantes e mantenedores do projeto
+- Bugs são rastreados através de [Github issues](https://guides.github.com/features/issues/) verifique que nenhuma issue foi criada por outro usuário reportando o mesmo erro.
+- Crie uma issue explicando o problema e adicionando novas informações detalhadas que ajudem os mantenedores do projeto a reproduzir o problema
```

Comment:
Excelente :+1:

Reply:
Boa! Talvez, a partir disso, a gente possa criar uma _issue_ para propor um [_template_](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#creating-issue-templates) ; )